Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM dockerfile/java:oracle-java7
+FROM java:openjdk-7-jre
```

Comment:
Would the `openjdk-7-jre-alpine` version work? If it does, that would make the image +/- 90MB smaller